### ISSUE TITLE
Rich Text fails with v7.1.0 - renders as "OfficeOpenXml.Style.ExcelRichTextCollection"

### ISSUE THREAD

**Issue by Sebazzz** [EPPlusSoftware/EPPlus#1387]
### EPPlus usage

Personal use

### Environment

Windows

### Epplus version

7.1.0

### Spreadsheet application

Excel

### Description

This code:

```C#
        ExcelRange headerRange = worksheet.Cells[1,1,1, 5];
        headerRange.Merge = true;
        
        ExcelRichTextCollection richText = headerRange.RichText;
        richText.Add("Hello");
        richText.Add(" ");
        ExcelRichText orgNameText = richText.Add("Peter Pan");
        orgNameText.Color = Color.Red;

```

In v7.0.10 this renders the table cell as expected.

In v7.1.0 it renders "OfficeOpenXml.Style.ExcelRichTextCollection" in the table cell.

**Comment by swmal** [comment]
Fix for this will be included in the next version

**Comment by swmal** [comment]
Fix provided in v 7.1.1
